test.adapters.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh -xe
 
 for pkg in $(find . -type d -name e2e | grep -v vendor | grep -v ".git"); do
-    time go test $pkg -v
+    time go test $pkg -v -count=1
```

Comment:
why did you even leave this file here if you run tests directly from `docker/test/test.sh`?

Reply:
I have a monday task to clean up the entire folder structure so it's nice and tidy. for the meanwhile I don't intend to delete this file until we sit with @netoneko and other contributors and understand who uses which of our fleet of `test` bash scripts. But for the meanwhile it seemed essential to at least make sure whoever uses this to test something isn't fooled by Go's test cache.

Reply:
Do you know with a 100% certainty that we should delete this bash script ?

Reply:
I think it's there solely for the e2e to run